iDiff/cmd/root.go

```diff
@@ -27,35 +27,44 @@ var RootCmd = &cobra.Command{
 			glog.Error(err.Error())
 			os.Exit(1)
 		}
+
 		utils.SetDockerEngine(eng)
-		image1, err := utils.ImagePrepper{args[1]}.GetImage()
+
+		img1Arg := args[len(args)-2]
```

Comment:
What happens if someone mistakenly puts the image args after the differs in the command? Can we maybe have the differs be specified as flags? i.e. you would pass `-p` or `--pip` for the pip differ, etc. Libraries like Cobra should be able to handle positional args by default.